settings/Localization.setting.php

```diff
@@ -528,4 +528,27 @@
     'help_text' => 'If a contact is created with no language this setting will determine the language data (if any) to save.'
     . 'You may or may not wish to make an assumption here about whether it matches the site language',
   ],
+  'favouriteContactCountries' => [
+    'group_name' => 'Localization Preferences',
+    'group' => 'localization',
+    'name' => 'favouriteContactCountries',
+    'type' => 'Array',
+    'quick_form_type' => 'Element',
+    'html_type' => 'advmultiselect',
+    'html_attributes' => [
+      'size' => 5,
+      'style' => 'width:150px',
+      'class' => 'advmultiselect',
+    ],
+    'default' => [],
+    'add' => '5.31',
+    'title' => ts('Favourite Countries'),
```

Comment:
If it's spelled as "Favourite" here in the title then there's no way for the traunslatioun systeum to output a US spelling.

Or another option is use a word like "Pinned". Or it's a bit long but "Countries Pinned to Top of Selection Lists" is accurate.

Also the `add` would currently be 5.33.

Reply:
@demeritcowboy , i changed label to `Pinned Countries` and corrected version too.